src/__mocks__/connectkit.tsx

```diff
@@ -0,0 +1,11 @@
+type MockConnectKitProviderProps = {
+  children: React.ReactNode;
+};
+
+const MockConnectKitProvider: React.FC<MockConnectKitProviderProps> = ({
+  children,
+}) => {
+  return <div className="mock-connectkit-provider">{children}</div>;
+};
+
+export { MockConnectKitProvider as ConnectKitProvider };
```

Comment:
Seems easier to just rename the mock, and export it directly?

Reply:
Done